Sources/MongoSwift/MongoCollection+FindAndModify.swift

```diff
@@ -383,6 +391,18 @@ private class FindAndModifyOptions {
         }
     }
 
+    fileprivate func setSession(_ session: ClientSession?) throws {
+        guard let session = session else {
+            return
+        }
+        var doc = Document()
+        try session.append(to: &doc)
+
+        guard mongoc_find_and_modify_opts_append(self._options, doc.data) else {
```

Comment:
this is a very silly api provided by libmongoc

Reply:
😞